lara-typescript/src/section-authoring/api/lara-api-provider.ts

```diff
@@ -0,0 +1,88 @@
+import {
+  IPage, PageId, ISection,
+  APIPageGetF, APIPagesGetF, IAuthoringAPIProvider,
+  ICreatePageItem
+} from "./api-types";
+
+const APIBase = "/api/v1";
+
+export const getLaraPageAPI = (host: string = "", activityId: string): IAuthoringAPIProvider => {
```

Comment:
It is kind of weird that the first parameter to `getLaraPageAPI` is optional and has a default value but the second one is required.